### ISSUE TITLE
Support Key-Based encryption (not from keys derived from a password)

### ISSUE THREAD

**Issue by LookingSharp** [TGIO/RNCryptorNative#1]
Thanks for sharing this @TGIO!

I use RNCryptor on iOS, and the iOS version supports encrypting directly from the keys, without having to pay for PBKDF2.  In the readme he covers it in the [Key-Based Encryption](https://github.com/RNCryptor/RNCryptor#advanced-usage) section.

In swift, key based encryption code looks like this - 

```
let _keyEncryption: NSData
let _keyHMAC: NSData

let encryptor = RNCryptor.EncryptorV3(encryptionKey: _keyEncryption, hmacKey: _keyHMAC)
let encryptedData = encryptor.encryptData(clearData)
```

It would be great if RNCryptorNative supported just using the keys.

**Comment by cs8** [comment]
rncryptor.encrypt(text ,password) 
but AES256JNCryptor is 
jncryptor.encryptData(text.getbytes(),password.toCharArray()) 
the result are not the same , And how to match it？

**Comment by getsadzeg** [comment]
You mean both encryptionKey and hmacKey?

**Comment by rnapier** [comment]
Correct. The v3 format requires that you provide both an encryption and hmac key. It will generate both for you from a password, but in the key mode, you need to provide both.

**Comment by getsadzeg** [comment]
Yes. I think it's gonna be written in native, C++.

**Comment by erawhctim** [comment]
Is there any update on the progress on this issue? Any open/in-progress pull requests? Any way to help?
